catlas/config_validation.py

```diff
@@ -180,23 +185,57 @@ def validate_folder_exists(field, value, error):
         "required": False,
         "type": "list",
         "schema": {
-            "type": "dict",
-            "schema": {
-                "checkpoint_path": {
-                    "type": "string",
-                    "check_with": validate_path_exists,
-                    "regex": ".*.pt",  # cerberus doesn't understand re "$"; requires full match
+            "anyof": [
+                {
+                    "type": "dict",
+                    "schema": {
+                        "checkpoint_path": {
+                            "type": "string",
+                            "check_with": validate_path_exists,
+                            "regex": ".*.pt",  # cerberus doesn't understand re "$"; requires full match
+                        },
+                        "gpu": {"type": "boolean", "required": True},
+                        "label": {
+                            "required": True,
+                            "type": "string",
+                        },
+                        "number_steps": {
+                            "type": "integer",
+                        },
+                        "batch_size": {"type": "integer"},
+                        "gpu_mem_per_sample": {"type": "float"},
+                        "step_type": {"allowed": ["inference"], "required": True},
+                    },
                 },
-                "gpu": {"type": "boolean", "required": True},
-                "label": {
-                    "required": True,
-                    "type": "string",
+                {
+                    "type": "dict",
+                    "schema": {
+                        "step_type": {
+                            "allowed": ["filter_by_adsorption_energy_target"],
+                            "required": True,
+                        },
+                        "target_value": {"type": "float", "required": True},
+                        "range_value": {"type": "float"},
```

Comment:
Seems maybe more intuitive to require a range value? A default value of 0.5 is super high for hydrogen for example

Reply:
Makes sense, changed!